bin/DMHS_ATS_configurable

```diff
@@ -45,13 +39,17 @@ def cmdline():
                         help="Port number for webserver")
     parser.add_argument("--send_efd_message", action="store_true", default=False,
                         help="Send and EDF message as well?")
+    parser.add_argument("--log", action="store", default='INFO',
+                         choices=["DEBUG","INFO","WARNING","ERROR","CRITICAL"],
```

Comment:
There is a flake8 violation here since you need a space after the comma.  Please set up continuous integration for PRs so that flake8 is automatically run.  I can help you with this.

Reply:
Thanks for catching that. Yes, I'd like to setup continuous integration.

Reply:
Sorry. I was doing a review but got distracted by a side conversation and ended up pushing "Add Single comment" all the time.